File: OCH_Tao/ABC/A/246.js
const input = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
const [x1, y1] = input[0].split(" ");
const [x2, y2] = input[1].split(" ");
const [x3, y3] = input[2].split(" ");
let l = [];
if (x1 === x2) {
  l.push(x3);
} else if (x2 === x3) {
  l.push(x1);
} else {
  l.push(x2);
}
if (y1 === y2) {
  l.push(y3);
} else if (y2 === y3) {
  l.push(y1);
} else {
  l.push(y2);
}
console.log(l.join(" "));

File: OCH_Tao/ABC/A/246.py
X1,Y1 = input().split()
X2,Y2 = input().split()
X3,Y3 = input().split()
l = []
if X1==X2:
  l.append(X3)
elif X2==X3:
  l.append(X1)
else:
  l.append(X2)
if Y1==Y2:
  l.append(Y3)
elif Y2==Y3:
  l.append(Y1)
else:
  l.append(Y2)
print(" ".join(l))